bin/build_website

```diff
@@ -22,11 +23,13 @@ rm -rf "${DEST_SITE_DIR}"
 mkdir -p "${DEST_SITE_DIR}"
 
 echo "Cleaning up generated files..."
-rm -rf $DOCS_DIR/generated/*.html
+rm -rf "$DOCS_DIR"/generated/*.html
 
 echo "Reading and generating godoc pages..."
-cat $DOCS_DIR/godoc_packages.txt | while read package; do
-  if [[ "${package}" =~ ^\#.* ]] || [[ "${package}" == "" ]]; then
+while read -r package "$DOCS_DIR/godoc_packages.txt"; do
```

Comment:
it seems to be hanging here when I run this script

Reply:
Fixed.